app/controllers/application_controller.rb

```diff
@@ -82,6 +82,17 @@ def request_event
     @request_event ||= RequestEvent.new(request, response, session, current_jobseeker, current_publisher_oid)
   end
 
+  def trigger_feedback_provided_event
+    feedback_data = feedback_params.each { |key, value|
```

Comment:
I believe you need `map` here not `each`

or if you want something a bit more rubish you can do:

```
feedback_params.each_with_object({}) do |(key, value), params| 
   %w[jobseeker_id publisher_id].include?(key) ? params["anonymised_#{key}"] = StringAnonymiser.new(value) :  params[key] = value }
end
```

Reply:
feedback_params doesn't have each_with_object so I have changed this to feedback_params.to_h.each_with_object

`undefined method `each_with_object' for #<ActionController::Parameters:0x000055904ae666c8>`